releasenotes/notes/magnum-capi-driver-2dd8cf6e68a1919c.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Adds the new capi driver into the magnum container
```

Comment:
Nit: I feel this should really link to your docs on how to install the magnum capi driver? Did we get that merged anywhere yet?